systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -671,6 +671,8 @@ void setupEnvironment() throws Exception {
         list.add(CruiseControlUtils.CRUISE_CONTROL_METRICS_TOPIC);
         list.add(CruiseControlUtils.CRUISE_CONTROL_MODEL_TRAINING_SAMPLES_TOPIC);
         list.add(CruiseControlUtils.CRUISE_CONTROL_PARTITION_METRICS_SAMPLES_TOPIC);
+        list.add("strimzi-store-topic");
+        list.add("strimzi-topic-operator-kstreams-topic-store-changelog");
```

Comment:
Are these the names of the topics? Shouldn't they have the `__` prefix? Also, would it make sense to start with the same name? E.g. `strimzi-topic-operator-store` instead of `strimzi-store-topic` when the other topic is named `strimzi-topic-operator...`?

Reply:
This was already a long discussion ... but sure, we can always change it?
@ppatierno @tombentley wdyt? ^